### DISCUSSION TITLE
Smart naming with sequential number?

### DISCUSSION THREAD

**Discussion by mariop486** [aclap-dev/video-downloadhelper#1532] [Help and Q&A]
### Your question

Hello everyone,
Is it possible in VDH 9 to smartnaming file adding a sequential number? or the hour (with seconds)?
Thanks !

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "used_history_button": true,
    "download_directory": "/Users/mario/Desktop/Video DownloadHelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/mario/Desktop"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": true,
        "low_quality": true,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "yt_warning": false,
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 59,
    "never_show_successfull_dl_message": true,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by Mattmjs** [comment]
Probably not possible, smartnaming is designed to take a name from the page you're on. What you could do is use Download as.
This gives you a dialog where you can make the name how you want before downloading.

**Reply by rashid0715** [reply]
can we at least pick and append the playlist index at the start of video name?

**Comment by lucasvperini** [comment]
+1
